src/rf/js/src/home/components/processing-block.js

```diff
@@ -17,7 +17,7 @@ var LayerStatusComponent = React.createBackboneClass({
             chunkClass = this.pendingClass,
             mosaicClass = this.pendingClass,
             completeClass = this.pendingClass,
-            actionLinks = (<a className="text-danger">Cancel</a>),
+            actionLinks = (<a href="javascript:;" onClick={this.deleteLayer} className="text-danger">Cancel</a>),
```

Comment:
It feels wrong to create a "non-clickable" link and then use CSS to make it visually appear clickable. This is not an unconventional design pattern.